src/gpu/PsyX_GPU.cpp

```diff
@@ -1298,6 +1298,12 @@ int ParsePrimitive(uintptr_t primPtr)
 		DR_PSYX_TEX* drtex = (DR_PSYX_TEX*)pTag;
 		break;
 	}
+	case 204: //TODO: rickomax - This seems a bit wrong, the tpage code is being stored in the len field, and enabling PG_XP bugs font display. I also don't know where dtd and dfe are
+		{
+		DR_TPAGE* drtPage = (DR_TPAGE*)pTag;
+		activeDrawEnv.tpage = drtPage->len;
```

Comment:
DR_TPAGE is handled below as DR_ENV subset. 0xCC means something is wrong with OT